human-viz-bridge/README.md

```diff
@@ -1,9 +1,69 @@
 ## Tools for interfacing HDE module architecture with ROS RViz.
 
-In order to launch urdf models, possible related publishers and rviz itself, launch files included in misc folder have to be employed.
-To launch these files use this command (each time for each launch file):
+In order to visualize but also to assess the results of the HDE module architecture, some YARP modules that take as input Xsens system, human state, and human dynamics data and publish them in the form of ros topics (so as to be read from Rviz) have been implemented.
 
+Information about RViz can be found here: [wiki.ros](http://wiki.ros.org/rviz).
```

Comment:
`wiki.ros` is not superclear as an reduced url, in this case I think the full url will be more clear: `[http://wiki.ros.org/rviz](http://wiki.ros.org/rviz)` .